Marlin/Marlin.ino

```diff
@@ -54,7 +54,6 @@
 #endif
 
 #if ENABLED(HAVE_TMC2130)
-  #include <SPI.h>
```

Comment:
Already included by `TMC2130Stepper.h`?

Reply:
Yep. Also the Archim2 uses software SPI and the library handles it according to flags set by the core.